docs/Contact_Candidate.html

```diff
@@ -15,4 +15,4 @@
="0 0 16 16" class="bi bi-arrow-right-short" fill=currentColor xmlns=http://www.w3.org/2000/svg> <path fill-rule=evenodd d="M4 8a.5.5 0 0 1 .5-.5h5.793L8.146 5.354a.5.5 0 1 1 .708-.708l3 3a.5.5 0 0 1 0 .708l-3 3a.5.5 0 0 1-.708-.708L10.293 8.5H4.5A.5.5 0 0 1 4 8z"/> </svg> <a href=#candidate_district onclick="anchorLink('candidate_district')">candidate_district</a></div><span class="badge badge-dark value-type">Type: string or null</span><br> <span class=pattern-value id=candidate_district_pattern>Must match regular expression: <code>^[0-9]{2}$</code></span> </div> </div> </div> </div> <div class=accordion id=accordiontelephone> <div class=card> <div class=card-header id=headingtelephone> <h2 class=mb-0> <button class="btn btn-link property-name-button" type=button data-toggle=collapse data-target=#telephone aria-expanded aria-controls=telephone onclick="setAnchor('#telephone')"><span class=property-name>telephone</span></button> </h2> </div> <div id=telephone class="collapse property-definition-div" aria-labelledby=headingtelephone data-parent=#accordiontelephone> <div class="card-body pl-5"> <div class=breadcrumbs>root <svg width=1em height=1em viewbox="0 0 16 16" class="bi bi-arrow-right-short" fill=currentColor xmlns=http://www.w3.org/2000/svg> <path fill-rule=evenodd d="M4 8a.5.5 0 0 1 .5-.5h5.793L8.146 5.354a.5.5 0 1 1 .708-.708l3 3a.5.5 0 0 1 0 .708l-3 3a.5.5 0 0 1-.708-.708L10.293 8.5H4.5A.5.5 0 0 1 4 8z"/> </svg> <a href=#telephone onclick="anchorLink('telephone')">telephone</a></div><span class="badge badge-dark value-type">Type: string or null</span><br> <span class=pattern-value id=telephone_pattern>Must match regular expression: <code>^\+\d{1,3} \d{10}$</code></span> </div> </div> </div> </div> <div class=accordion id=accordioncountry> <div class=card> <div class=card-header id=headingcountry> <h2 class=mb-0> <button class="btn btn-link property-name-button" type=button data-toggle=collapse data-target=#country aria-expanded aria-controls=country onclick="setAnchor('#country')"><span class=property-name>country</span> <span class="badge badge-warning required-property">Required</span></button> </h2> </div> <div id=country class="collapse property-definition-div" aria-labelledby=headingcountry data-parent=#accordioncountry> <div class="card-body pl-5"> <div class=breadcrumbs>root <svg width=1em height=1em viewbox="0 0 16 16" class="bi bi-arrow-right-short" fill=currentColor xmlns=http://www.w3.org/2000/svg> <path fill-rule=evenodd d="M4 8a.5.5 0 0 1 .5-.5h5.793L8.146 5.354a.5.5 0 1 1 .708-.708l3 3a.5.5 0 0 1 0 .708l-3 3a.5.5 0 0 1-.708-.708L10.293 8.5H4.5A.5.5 0 0 1 4 8z"/> </svg> <a href=#country onclick="anchorLink('country')">country</a></div><span class="badge badge-dark value-type">Type: string</span><br> </div> </div> </div> </div> <footer> <p class=generated-by-footer>Generated using <a href=https://github.com/coveooss/json-schema-for-humans>json-schema-for-humans</a> on 2022-10-13 at 13:22:20 -0400</p> </footer></body> </html>
```

Comment:
There are no changes to CONTACT_COMMITTEE.json included in this pr.  Where they supposed to be committed?

Reply:
The contact committee JSON has what I think is the correct regex so no changes were necessary. Again, this assumes Aurelia confirms this is the correct pattern. I am guessing it is based on the ZenHub ticket for the committee id for the committee contact. https://app.zenhub.com/workspaces/fecfile-online-619e578e68408b001c831251/issues/fecgov/fecfile-web-app/205